frontend/talentsearch/src/js/components/Home/partials/Featured/Featured.tsx

```diff
@@ -4,54 +4,86 @@ import { useIntl } from "react-intl";
 import { imageUrl } from "@common/helpers/router";
 
 import Heading from "@common/components/Heading";
+import { getLocale } from "@common/helpers/localize";
 import TALENTSEARCH_APP_DIR from "../../../../talentSearchConstants";
 
 import Block from "./Block";
 
 // Create the page component
 const Featured = () => {
   const intl = useIntl();
+  const locale = getLocale(intl);
+
+  const iapEmail = {
+    subject: encodeURIComponent(
+      "I'm interested in hiring Indigenous IT apprentices for my team",
+    ),
+    body: encodeURIComponent(
+      "I discovered the Indigenous Apprenticeship Program on talent.canada.ca and I’d like to learn more about how I can hire apprentices to my team.",
```

Comment:
This needs to be wrapped in `intl.formatMessage` so it can be translated.

Reply:
Updated in [clean up copy (whitespace, quotes)](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4442/commits/a739b51e2dc86a2cf4f4b20de407ec2995705665)